docs/se/guides/07_db_client.adoc

```diff
@@ -0,0 +1,604 @@
+----
+
+Instead of sending the `INFO` content of the targeted book, the `getJsonBook` method send the whole row of the
+database as a `JsonObject`.
+
+[source,java]
+.Add addBook to the LibraryService:
+----
+private void addBook(ServerRequest serverRequest, ServerResponse serverResponse) {
+        String bookName = serverRequest.path().param("name");
+
+        serverRequest.content()
+                .as(String.class)
+                .thenAccept(newValue -> {
+                    dbClient.execute(exec -> exec.createNamedInsert("insert-book")
+                            .addParam("name", bookName)     // <1>
+                            .addParam("info", newValue)
+                            .execute())
+                            .thenAccept(count -> serverResponse.status(Http.Status.CREATED_201).send())     // <2>
+                            .exceptionally(serverResponse::send);
+                });
+}
+----
+<1> The SQL statement requires the book name and its information. They are provided with `addParam` method.
+<2> A new book was added to library, so a Http 201 code is returned.
+
+When a user adds a new book, it uses Http PUT method where the book name is in the url and the information in the
+request content. To catch this content, the information is retrieved as a string and then the DB Client execute the
+`insert-book` script to add the book to the library. It requires two parameters, the book name and information which are
+passed to the dbClient thanks to `addParam` method. A Http 201 is sent back as a confirmation if no exception is thrown.
+
+[source,java]
+.Add deleteBook to LibraryService:
+----
+private void deleteBook(ServerRequest serverRequest, ServerResponse serverResponse) {
+    String bookName = serverRequest.path().param("name");
+
+    dbClient.execute(exec ->  exec.namedDelete("delete-book", bookName))     // <1>
+                        .thenAccept(count -> serverResponse.status(Http.Status.NO_CONTENT_204).send())   // <2>
+                        .exceptionally(serverResponse::send);
+}
+----
+<1> Execute SQL script from application.yaml to remove a book from the library by its name.
+<2> The required book was removed, so a Http 204 is sent.
+
+To remove a book from the library, use the "delete-book" script in the way than previously. If the book is removed
+successfully, a Http 204 is sent back.
+
+== Build and Run the Library application
+
+The application is ready to be built and ran.
+
+[source,bash]
+.Run the following to build the application:
+----
+mvn package
+----
+
+Note that the tests are passing as the GreetService process was not modify. During this guide, we only added independent
+new content to the existing application.
+Make sure H2 is running and start the helidon quickstart by this command:
+
+[source,bash]
+.Run the application
+----
+java -jar target/helidon-quickstart-se.jar
+----
+
+Once the application start, check the table LIBRARY is created into H2 database. To do so, go to the
```

Comment:
start -> starts
into H2 -> in the H2
and it will appear -> and it should appear